TidepoolOnboardingKitUI/TidepoolOnboardingUI.swift

```diff
@@ -19,7 +19,7 @@ public final class TidepoolOnboardingUI: OnboardingUI {
         return TidepoolOnboardingUI()
     }
 
-    public func onboardingViewController(cgmManagerProvider: CGMManagerProvider, pumpManagerProvider: PumpManagerProvider, serviceProvider: ServiceProvider, preferredGlucoseUnit: HKUnit, colorPalette: LoopUIColorPalette) -> (UIViewController & OnboardingViewController) {
-        return OnboardingRootViewController(cgmManagerProvider: cgmManagerProvider, pumpManagerProvider: pumpManagerProvider, serviceProvider: serviceProvider, preferredGlucoseUnit: preferredGlucoseUnit, colorPalette: colorPalette)
+    public func onboardingViewController(cgmManagerProvider: CGMManagerProvider, pumpManagerProvider: PumpManagerProvider, serviceProvider: ServiceProvider, displayGlucoseUnit: HKUnit, colorPalette: LoopUIColorPalette) -> (UIViewController & OnboardingViewController) {
```

Comment:
[nit] When are we going to update all of the plugin protocols to take `DisplayGlucoseUnitObservable` (rather than `HKUnit`) and remove the protocol requirement for `DisplayGlucoseUnitObserver` (which was `PreferredGlucoseUnitObserver`)? I was under the impression that was part of the purpose for this change.

Reply:
ah... yes. slipped my mind, but I'll do that as part of this update.